app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.pec_acm.moviedroid
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class ProfileActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_profile)
+    }
+}
```

Comment:
Please add the functionality to display the name, image and email of the user in the corresponding views. Make sure to use view binding for accessing the views.

Reply:
From which activity i can get this info?

Reply:
You can get the current user through this
```kt
val user = FirebaseAuth.getInstance().currentUser
```
and then access the `displayName` , `photoUrl` and `email`  from this `user` object 😇

An example of this:
https://github.com/PEC-CSS/MovieDroid/blob/b693a46cc030007abeb4bec2fd0b383b34902355/app/src/main/java/com/pec_acm/moviedroid/MainActivity.kt#L65-L71